### ISSUE TITLE
508-defect-3 [COGNITION]: Contact Us - Page should have a "Contact Us" breadcrumb to match top level navigation pattern

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#17987]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a user who relies on breadcrumbs to understand where I am in the VA.gov website, I would like to see a "Contact Us" breadcrumb link when I land on this page.

## Details

Looking at our other top level pages like Find Locations, we have a breadcrumb that reflects the page title. Screenshot of current and proposed below.

/cc @mnorthuis in case this pattern is incorrect or there's a better link name.

## Acceptance Criteria

- [ ] Breadcrumb update is made (if applicable)
- [ ] Ticket is closed with comment if no action or breadcrumb update is required

## Environment

* http://preview-prod.vfs.va.gov/preview?nodeId=8472 (must be on the network)

## WCAG or Vendor Guidance (optional)

* [WCAG 2AAA Location: Understanding SC 2.4.8](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-location.html)

### Current
![Screen Shot 2020-12-31 at 11 20 02 AM](https://user-images.githubusercontent.com/934879/103421402-becdc780-4b61-11eb-9cd0-2e93e734a4a4.png)

### Proposed
![Screen Shot 2020-12-31 at 11 20 36 AM](https://user-images.githubusercontent.com/934879/103421409-c42b1200-4b61-11eb-9570-2dfed5a88336.png)

**Comment by bethpottsVADEPO** [comment]
@RLHecht Would you check in with Mikki about this for her guidance.

**Comment by mnorthuis** [comment]
@bethpotts @RLHecht The proposed breadcrumb from @1Copenut is the correct structure.

**Comment by DanielleThierryUSDSVA** [comment]
Hi there. Just confirming that I've added a breadcrumb:

![image.png](https://images.zenhubusercontent.com/5e8b13016a4378a604f5e799/a5073257-6724-4a39-98dc-8b6a9b5db373)

**Comment by 1Copenut** [comment]
Thank you @DanielleThierryUSDSVA ! I"ll move this one to closed.
